README.md

```diff
@@ -27,6 +27,52 @@ const MyComponent = () => (
 
 ```
 
+### Importing Sass Modules
+
+Example using Webpack:
+
+```js
+module.exports = {
+  ...,
+  resolve: {
+    alias: {
+      "@Feathers": path.resolve(__dirname, "node_modules/srf-feathers/build/foundation")
```

Comment:
nitpick: damit man mehr als nur die Foundations benutzen kann, wär's wohl eher `srf-feathers/build`, nöd?

Reply:
Danke für das Feedback. Habe ich in https://github.com/mmz-srf/feathers/pull/31/commits/9da704e15fd9c8ea3462097049e2aefdf755756f gleich so angepasst und im Tweety auch so übernommen: https://github.com/mmz-srf/srf-comments/pull/354/commits/3269a4f4d08b7fdabb30f7739131aaa4a1b6b7d5

Reply:
Top, merci!